src/hle.c

```diff
@@ -352,6 +352,7 @@ static ucode_func_t try_normal_task_detection(struct hle_t* hle)
         return &jpeg_decode_OB;
     }
 
+    /* Resident Evil 2 */
```

Comment:
Thanks for those comments, we don't have many, but games or public rom names reaching a particular code place is always useful for later review.